### PR TITLE
Websocket implementation

### PR DESCRIPTION
**Pull request by gunjack-el**
Websocket implementation, all channels are supported as described in the Coinbase documentation (https://docs.cdp.coinbase.com/advanced-trade/docs/ws-channels/).

### CODE REVIEW

File: coinbaseadvanced/client.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 class AuthSchema(Enum):
     """
     Enum representing authetication schema:
-    https://docs.cloud.coinbase.com/advanced-trade-api/docs/auth#authentication-schemes
+    https://docs.cdp.coinbase.com/advanced-trade/docs/auth#authentication-schemes
```
Comment (KmiQ) [inline coinbaseadvanced/client.py:31]: thanks for the updates on the docs link.

**Review by KmiQ — APPROVED** [review]
Overall LGTM. I'd say let's get it merged and I will include it in the next week release. Thank you @gunjack-el!

**Comment by gunjack-el** [comment]
When will the new release be?

**Comment by gunjack-el** [comment]
> Overall LGTM. I'd say let's get it merged and I will include it in the next week release. Thank you @gunjack-el!

Hi, sorry to bother you, but I wanted to ask if any further confirmation or action is expected from my side? I'm not very familiar with GitHub. If there's anything needed from me, please let me know, and I'll take care of it right away.

**Comment by gunjack-el** [comment]
> Overall LGTM. I'd say let's get it merged and I will include it in the next week release. Thank you @gunjack-el!

Hi, sorry to bother you, but I wanted to ask if any further confirmation or action is expected from my side? I'm not very familiar with GitHub. If there's anything needed from me, please let me know, and I'll take care of it right away.

**Comment by KmiQ** [comment]
> > Overall LGTM. I'd say let's get it merged and I will include it in the next week release. Thank you @gunjack-el!
> 
> Hi, sorry to bother you, but I wanted to ask if any further confirmation or action is expected from my side? I'm not very familiar with GitHub. If there's anything needed from me, please let me know, and I'll take care of it right away.

Sorry, this was my bad. I was assuming you were gonna merge the PR but I realize I never make it clear. Thanks for bringing this back again @gunjack-el . I'm merging the PR and filing a new release. Apologies for the delay and inconvenience.

**Pull request merged**